docs/README.md

```diff
@@ -1136,25 +1135,6 @@ _Reference_
 - https://radiant-brushlands-42789.herokuapp.com/medium.com/better-programming/javascript-es2020-features-with-simple-examples-d301dbef2c37
 - https://www.freecodecamp.org/news/javascript-new-features-es2020/
 
-<br/> 
-
-## 17. Local storage vs Session storage
-
```

Comment:
Hey why did you remove this content?
Pls add it back and then I could merge the pr